config/filter.d/apache-common.conf

```diff
@@ -3,12 +3,31 @@
 
 [INCLUDES]
 
+before = common.conf
 # Load customizations if any available
 after = apache-common.local
 
 [DEFAULT]
 
-_apache_error_client = \[\] \[(:?error|\S+:\S+)\]( \[pid \d+(:\S+ \d+)?\])? \[client <HOST>(:\d{1,5})?\]
+# Apache logging mode:
+#   all - universal prefix (logfile, syslog)
+#   logfile - logfile only
+#   syslog - syslog only
+# Use `filter = apache-auth[logging=syslog]` to get more precise regex if apache logs into syslog (ErrorLog syslog).
+# Use `filter = apache-auth[logging=all]` to get universal regex matches both logging variants.
+logging = logfile
+
+# Apache logging prefixes (date-pattern prefix, server, process etc.):
+apache-prefix-syslog = %(__prefix_line)s
+apache-prefix-logfile = \[\]\s
+apache-prefix-all = (?:%(apache-prefix-logfile)s|%(apache-prefix-syslog)s)?
+
+# Setting for __prefix_line (only `logging=syslog`):
+_daemon = (?:apache\d*|httpd(?:/\w+)?)
```

Comment:
Apache has many options to configure everything, so whole log-format may be different.
We support only default formats. Another things should be then configured via own/custom filter...
Back to your question, you can set in to other thing (v0.10 only):
```INI
# jail.local
[apache-auth]
filter = apache-auth[logging=syslog, _daemon="my-apache"]
```
Or via apache-common.local:
```INI
[Definition]
_daemon = \S*
```